plugins/google_search/plugin.spec.yaml

```diff
@@ -43,29 +46,24 @@ actions:
         description: Number of results to retrieve to limit amount, default is 16
           and the search will allows return at least 16 results if available. This
           option seems to have odd behavior
+        example: 16
         default: 16
         required: true
       num:
         title: Number
         type: integer
         description: Number of results per page, default is 10
+        example: 10
         default: 10
         required: true
-      only_standard:
-        title: All
-        type: boolean
-        description: Only returns the standard results from each page. If false, it
-          returns every possible link from each page, except for those that point
-          back to Google itself
-        default: false
-        required: true
       pause:
         title: Pause
         type: float
         description: Lapse to wait between HTTP requests. A lapse too long will make
           the search slow, but a lapse too short may cause Google to block your IP.
           Your mileage may vary
         default: 1
```

Comment:
Would default here be 1.0 or just one, since it is type `float` ?

Reply:
I don't think it should impact but having it as 1.0 seems like the better option. have updated this now.